.github/workflows/ci.yml

```diff
@@ -44,6 +44,11 @@ jobs:
             django-version: 5.0
           - python-version: 3.9
             django-version: 5.0
+          # Unsupported Python versions for Django 5.1
+          - python-version: 3.8
+            django-version: 5.1
+          - python-version: 3.9
+            django-version: 5.1
```

Comment:
I think you’ll need to add 5.1 in django-version list in the matrix, line 32.

Reply:
Thank you for your comment. It was my mistake. I appreciate you pointing it out. I've made the correction and pushed a new commit https://github.com/dfunckt/django-rules/pull/193/commits/ce5877c9d700b6b039486fab50a220263c022839